conans/model/settings.py

```diff
@@ -172,7 +172,8 @@ def __init__(self, definition=None, name="settings", parent_value="settings"):
                       for k, v in definition.items()}
 
     def serialize(self):
-        return {k: v.value for k, v in self._data.items()}
+        # Returning all the information available, including the sub-settings
+        return {k: v for k, v in self.items()}
```

Comment:
This was done on purpose to not keep using the legacy bit of mess that is behind ``self.items()``, that calls ``self.fields()``,  ``self.values_list``.

Reply:
Ok, let me have a look at it.